free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL would help users learn more about flavor pairings and make suggestions if given specific foods. The users would be able to figure out foods or beverages that would pair well with a given item. For example, if a person were hosting a dinner party and didn't know what to serve with the main dish, they could receive suggestions about flavors or dishes that would be good accompaniments. 
+
```

Comment:
This seems like a neat idea. I personally don't know anything about flavor pairings, so if I'm missing something from a recipe it ends up being a disaster.